src/navigator/DrawerTopBar.tsx

```diff
@@ -20,6 +25,7 @@ interface Props {
 
 function DrawerTopBar({ leftElement, middleElement, rightElement, scrollPosition, testID }: Props) {
   const navigation = useNavigation()
+  const useTabNavigator = getFeatureGate(StatsigFeatureGates.USE_TAB_NAVIGATOR)
```

Comment:
this works for now, but ideally we should make a separate component for the new header, which we could get to in ACT-1108, maybe leave a TODO?

Reply:
Added Todo's in d7ba1693653b9e9699a3e790e12190a5a7ce6efd!